### ISSUE TITLE
Broken `reverse_each` autocorrect

### ISSUE THREAD

**Issue by ignat-z** [testdouble/standard#230]
It seems like `standard` produces incorrect code for multiline `reverse.each`.
Cop: `Performance/ReverseEach: Use 'reverse_each' instead of 'reverse.each'.`

Minimal source to reproduce:

```ruby
[1, 2, 3]
  .reverse
  .each do |idx|
    puts idx
  end
```

```ruby
# standardrb examplex.rb --fix
[1, 2, 3]
  .reverse
_each do |idx|
  puts idx
end
```

```ruby
# rubocop -A examplex.rb
# frozen_string_literal: true

[1, 2, 3]
  .reverse
  .each do |idx|
    puts idx
  end
```

```
        Ruby version: 2.7.1
     RuboCop version: 1.4.2
    Standard version: 0.10.2
Standard config file: [No file found]
```

**Comment by koic** [comment]
This is a bug of RuboCop Performance. It will be fixed by https://github.com/rubocop-hq/rubocop-performance/pull/201. Thank you for your feedback.

**Comment by koic** [comment]
This issue has been resolved by RuboCop Performance 1.9.2. Thank you 🌅
